### ISSUE TITLE
Broken links for f# Freya, WebApi, Frank

### ISSUE THREAD

**Issue by wallymathieu** [panesofglass/TodoBackendFSharp#9]
Should someone setup a new instance with a new URL for the todobackend in F#?
See issue:
https://github.com/TodoBackend/todo-backend-site/issues/174#issuecomment-573799711

**Comment by panesofglass** [comment]
Yes, I let my Azure subscription lapse, so this is no longer running. All the versions are woefully out of date, as well.

**Comment by wallymathieu** [comment]
Perhaps something we could get F# org to host?
